opentelemetry/proto/trace/v1/trace.proto

```diff
@@ -329,5 +343,26 @@ enum SpanFlags {
   // Bits 0-7 are used for trace flags.
   SPAN_FLAGS_TRACE_FLAGS_MASK = 0x000000FF;
 
-  // Bits 8-31 are reserved for future use.
+  // Bits 8 and 9 are used to indicate that the parent span or link span is remote.
+  // See SpanFlagsContextIsRemote for the valid values.
+  SPAN_FLAGS_CONTEXT_IS_REMOTE_MASK = 0x00000300;
+
+  // Bits 10-31 are reserved for future use.
+}
+
+// SpanFlagsContextIsRemote represents constants used to interpret
+// bits 8 and 9 of the Spans.flags or Link.flags field. These two bits
+// represent whether the parent or link is remote. Bit 8 represents whether
+// the field has been set and bit 9 represents whether the parent or link is
+// remote.
+// Once the bits have been read using SPAN_FLAGS_CONTEXT_IS_REMOTE_MASK,
+// the following constants can be used to interpret them.
+enum SpanFlagsContextIsRemote {
```

Comment:
Confused why duplicate the `SPAN_FLAGS_CONTEXT_IS_REMOTE_MASK` value? Can we only have mask values in the `SpanFlags` enum?